go/service/delegate_ui.go

```diff
@@ -47,3 +47,14 @@ func (d *DelegateUICtlHandler) RegisterUpdateUI(_ context.Context) error {
 	d.G().UIRouter.SetUI(d.id, libkb.UpdateUIKind)
 	return nil
 }
+
+func (d *DelegateUICtlHandler) RegisterRekeyUI(_ context.Context) error {
+	d.G().UIRouter.SetUI(d.id, libkb.RekeyUIKind)
+
+	// Let Gregor related code know that a RekeyUI client
```

Comment:
I agree. So is there no way then to output to error log in the case that electron doesn't connect? 

Actually, i think the answer is simple -- to attach another handler that just output rekey notifications to the error log, which will happen whether or not electron is active.